lib/agoneum/account/user.ex

```diff
@@ -4,6 +4,12 @@ defmodule Agoneum.Account.User do
   import Ecto.Changeset
   alias Agoneum.Account.User
 
+  @required_fields ~w(email name)a
+  @optional_fields ~w(active admin password)a
```

Comment:
I wonder if, for now, we can let this slide and add a `TODO`. I 100% agree with Matt, but I also wonder if we should think more deeply about roles instead of just having this a binary (thinking longterm there)